azure-pipelines.yml

```diff
@@ -45,12 +45,9 @@ steps:
   displayName: 'Build QEMU q35 board for x86'
 - script: |
     set -e
-    git clone https://github.com/qemu/qemu && cd qemu && git checkout v5.1.0
-    mkdir build-riscv64 && cd build-riscv64
-    ../configure --target-list=riscv64-softmmu
-    make -j16
-    sudo ln -s $PWD/riscv64-softmmu/qemu-system-riscv64 /usr/bin/
-    sudo ln -s $PWD/qemu-img /usr/bin/
+    git clone https://github.com/gevorgyana/simbin.git && cd simbin
+    ./build.sh
+    sudo cp qemu/build-riscv64/riscv64-softmmu/qemu-system-riscv64 /usr/bin
```

Comment:
For now, this builds qemu from source, done here to prove that it works. When this oreboot/simbin#1 is merged, we will just `git checkout https://github.com/oreboot/simbin.git` and `sudo cp` the binary to `usr/bin`. Anyway we should probably wait for this oreboot/simbin#1.